crates/iota-names/src/config.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+
+use iota_types::{
+    TypeTag,
+    base_types::{IotaAddress, ObjectID},
+    supported_protocol_versions::Chain,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::Domain;
+
+pub const LEAF_EXPIRATION_TIMESTAMP: u64 = 0;
+pub const DEFAULT_TLD: &str = "iota";
+pub const ACCEPTED_SEPARATORS: [char; 2] = ['.', '*'];
+pub const IOTA_NEW_FORMAT_SEPARATOR: char = '@';
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "kebab-case")]
+pub struct IotaNamesConfig {
+    pub package_address: IotaAddress,
+    pub object_id: ObjectID,
+    pub payment_package_address: IotaAddress,
+    pub registry_id: ObjectID,
+    pub reverse_registry_id: ObjectID,
+    pub subdomain_proxy_package_id: ObjectID,
```

Comment:
Can't we add some comments to those fields? For me they are not really obvious if I look at the IOTA Name stuff for the first time.